ocs_ci/deployment/helpers/storage_class.py

```diff
@@ -37,7 +40,7 @@ def get_storageclass() -> str:
     if customized_deployment_storage_class:
         storage_class = customized_deployment_storage_class
     else:
-        storage_class = DEFAULT_STORAGE_CLASS_MAP[platform]
+        storage_class = DEFAULT_STORAGE_CLASS_MAP.get(platform)
```

Comment:
This will return None if the platform is not present in DEFAULT_STORAGE_CLASS_MAP. So change to None is not actually needed in the dict. But considering the usage if needed in other places, this looks good.